source/_integrations/govee_ble.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Govee BLE
+description: Instructions on how to integrate Govee BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_bluetooth: true
+ha_release: 2022.8
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@bdraco'
+ha_domain: govee_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [Govee](https://www.govee.com/) BLE devices into Home Assistant.
+
+## Supported devices
```

Comment:
I'd put this in the order:

- Configuration
- Paragraph about setting up bluetooth to discover devices.
- Supported devices.

Reply:
a4be232a18bcfe3dcaa34ccf154369ab0f0828ea